src/Sarif.UnitTests/Sarif.UnitTests.csproj

```diff
@@ -30,80 +30,6 @@
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\MinimumWithTwoRuns.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\NestedFiles.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\NotificationExceptionWithStack.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\OneRunWithBasicInvocation.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\OneRunWithFiles.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\OneRunWithInvocationAndNotifications.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\OneRunWithLogicalLocations.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\OneRunWithNotificationsButNoInvocations.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\OneRunWithRules.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\RestoreFromPropertyBag.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\TwoResultsWithFixes.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\Inputs\UriBaseId.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\BasicResult.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\CodeFlows.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\Minimum.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\NestedFiles.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\MinimumWithPropertiesAndTags.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\MinimumWithTwoRuns.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\NotificationExceptionWithStack.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\OneRunWithBasicInvocation.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\OneRunWithFiles.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\OneRunWithInvocationAndNotifications.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\OneRunWithLogicalLocations.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\OneRunWithNotificationsButNoInvocations.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\OneRunWithRules.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\RestoreFromPropertyBag.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\TwoResultsWithFixes.sarif" />
-    <None Remove="TestData\SarifVersionOneToCurrentVisitor\ExpectedOutputs\UriBaseId.sarif" />
-  </ItemGroup>
```

Comment:
>ItemGroup [](start = 4, length = 9)

This was pointless. `None` items are those that "play no role in the build process". All of these files now appear as `EmbeddedResource` items. Marking them with `<None Remove="..." />` would have had the effect of making them not appear in the Solution Explorer -- _if_ they weren't mentioned elsewhere.

I think what happened is that we originally took all the `EmbeddedResource Include=` items and changed them to `None Include=` so that they still appeared in the project file. Then, as we restored the various tests, we _should_ have changed each `None` back to `EmbeddedResource`. Instead, we _added_ new `EmbeddedResource` items, and maked the `None` items as `Remove`.

Reply:
These edits were made by VS itself. You can adjust this behavior if you are absolutely certain there are no other implications. I'm not super motivated to stay on top of how the project system is maintaining things, however.

Reply:
Looks like a good edit to the project. 

---
In reply to: [251215559](https://github.com/Microsoft/sarif-sdk/pull/1233#discussion_r251215559) [](ancestors = 251215559)